charts/sequencer/values.yaml

```diff
@@ -180,12 +180,13 @@ upgrades:
     #       id: "1"
 
 sequencer:
+  logLevel: "info"
   abciUDS: true
   mempool:
     parked:
       maxTxCount: 200
   priceFeed:
-    enabled: true
+    enabled: false
```

Comment:
by default keep price feeds disabled, these will only be running in dusk-11 right now, and only useful on validator nodes